src/jacobi/mod.rs

```diff
@@ -188,7 +207,10 @@ mod tests {
 
     #[test]
     fn golub_welsch_5_alpha_1_beta_0() {
-        let (x, w) = GaussJacobi::nodes_and_weights(5, 1.0, 0.0);
+        let (x, w): (Vec<_>, Vec<_>) = GaussJacobi::new(5, 1.0, 0.0)
+            .into_iter()
+            .into_iter()
```

Comment:
Double into_iter()

Reply:
Maybe if I call it more times it goes faster? :D